pkg/distributor/ha_tracker_test.go

```diff
@@ -135,7 +136,7 @@ func TestHATrackerConfig_Validate(t *testing.T) {
 }
 
 // Test that values are set in the HATracker after WatchPrefix has found it in the KVStore.
-func TestWatchPrefixAssignment(t *testing.T) {
+func TestHATrackerWatchPrefixAssignment(t *testing.T) {
```

Comment:
All HA tracker tests now have `TestHATracker` prefix, so that we can run them easily.